src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -566,5 +538,20 @@ public void injectJenkinsToBitbucketCredentials() {
                 Guice.createInjector(new JenkinsToBitbucketCredentialsModule()).injectMembers(this);
             }
         }
+
+        @Override
+        public SCM newInstance(@Nullable StaplerRequest req, @Nonnull JSONObject formData) throws FormException {
+            BitbucketSCM instance = (BitbucketSCM) super.newInstance(req, formData);
+            
+            if (req != null) {
+                if (!req.getAncestors().isEmpty()) {
```

Comment:
Of slight note, I don't believe this is possible in a standard Jenkins environment (the Jenkins object should always be a root ancestor) but I can imagine the acceptance test harness might do things a bit differently so here just to be safe